src/services/api/actions/__tests__/getDrugSearchResults.test.js

```diff
@@ -1,13 +1,12 @@
 import { getDrugSearchResults } from '../index';
 
 describe('getDrugSearchResults action', () => {
-	test(`should match getDrugSearchResults action for drug "acetic acid"`, () => {
-		const drug = 'acetic acid';
+	test(`should match getDrugSearchResults action for drug "bevacizumab"`, () => {
```

Comment:
What exactly is this test for? This seems to be for the search page, correct???

Reply:
additionally, this is a test where you don't call functions, but you make sure that 
`acetic acid` becomes `acetic%20acid`. Using a one word term does not test that. (you know or searching for `(1-something,,2-blah)` or whatever complicated chemical name someone my autosuggest/search for.
@kate-mashkina ^^^

Reply:
This tests that the fetch action endpoint string for drug search matches what's expected. Yes, it would be related to the search page since that's where the call will eventually be used. This was added at the setup of the app framework to ensure the fetch call was wired right and works.

Reply:
Updated test to use string `acetic%20acid` inside expected fetch action endpoint